### DISCUSSION TITLE
Hide server admin password until clicked.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#438] [Suggestions]
Would it be possible to set the admin password as hidden until clicked to reveal? More of an issue with the new overview panel than the previous settings screen as before you had to scroll way down the page to expose it. Now if screen sharing it is immediately exposed when you navigate to a server.

**Comment by mvaessen** [comment]
Great suggestion! I actually had this on my todo list for the Game server QoL! Going to convert this into a development item right away
